<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewQuiz</class>
 <widget class="QDialog" name="CreateNewQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="CreateQuizOkbutton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Grade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateQuizCancelbutton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewQuizlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>New Quiz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EnterNamelineedit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>101</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChooseDirlineedit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>181</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChooseTestcaselineedit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>261</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChooseAnswerlineedit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>341</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EnterNamlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Quiz Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChooseDirlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Delivery Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChooseTestcaselabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Testcase File</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChooseAnswerlabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Answer File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseDirButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseTestcasebutton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChooseAnswerbutton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exectimeLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Execution Time </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
